lib/internal/process/promises.js

```diff
@@ -14,6 +14,8 @@ function getAsynchronousRejectionWarningObject(uid) {
 }
 
 function setupPromises(scheduleMicrotasks) {
+  var deprecationWarned = false;
```

Comment:
IMO, it would be nice change it to `let` while you are here, but LGTM either way.